src/objectExplorer/objectExplorerService.ts

```diff
@@ -229,7 +229,15 @@ export class ObjectExplorerService {
      */
     public sortByServerName(array: TreeNodeInfo[]): TreeNodeInfo[] {
         const sortedNodeArray = array.sort((a, b) => {
-            return (a.label as string).toLowerCase().localeCompare((b.label as string).toLowerCase());
+            if (typeof a.label === 'string' && typeof b.label === 'string') {
```

Comment:
to simplify the code, I would first get the string value for a and b and then do the compare